specifyweb/frontend/js_src/lib/components/DataModel/businessRules.ts

```diff
@@ -112,7 +109,8 @@ export class BusinessRuleManager<SCHEMA extends AnySchema> {
     fieldName: string & keyof SCHEMA['fields'],
     results: RA<BusinessRuleResult<SCHEMA>>
   ) {
-    const field = this.resource.specifyTable.strictGetField(fieldName);
+    const field = this.resource.specifyTable.getField(fieldName);
```

Comment:
Instead of this change, check if field is in `specialFields` and if so, don't do anything

Reply:
Is the change in b91ad63 sufficient?